common/extensions/api/brave_rewards.json

```diff
@@ -391,6 +391,12 @@
         "description": "Opens the Rewards panel in the active window.",
         "parameters": []
       },
+      {
+        "name": "openRewardsPanelToDeclareCountry",
```

Comment:
Naming: maybe `showRewardsSetup`?

Reply:
Addressed in https://github.com/brave/brave-core/pull/18717/commits/96ea7722bb70265f1f6b0cadd96fa5e73df23818.